admin-dev/themes/new-theme/js/components/grid/extension/link-row-action-extension.js

```diff
@@ -70,8 +70,16 @@ export default class LinkRowActionExtension {
         /*
          * Only search for cells with non clickable contents to avoid conflicts with
          * previous cell behaviour (action, toggle, ...)
+         *
+         * @todo: only td.clickable should be in this list,
+         *  you could remove all the rest and it would work,
+         *  we keep it in 1.7.7 for retrocompatibilty purpose,
+         *  just in case some people extend the template and won't have td.clickable
+         *  (only td.indentifier-type was removed)
+         *
+         * Columns are know specified as clickable in the grid definitions
          */
-        const clickableCells = $('td.data-type, td.identifier-type, td.badge-type, td.position-type', $parentRow)
+        const clickableCells = $('td.data-type, td.badge-type, td.position-type, td.clickable', $parentRow)
```

Comment:
Why keeping `td.data-type, td.badge-type, td.position-type`? If those cells are set "clickable" in the grid definition, why `td.clickable` wouldn't be enough?

Reply:
Why remove the `td.identifier-type` (there is a typo by the way)
I understand the issue when there is a checkbox, but it's not always the case